src/Wallabag/CoreBundle/Helper/Tools.php

```diff
@@ -118,4 +118,16 @@ public static function generateToken()
 
         return str_replace('+', '', $token);
     }
+
+    /**
+     * For a given text, we calculate reading time for an article.
+     *
+     * @param $text
+     *
+     * @return float
+     */
+    public static function getReadingTime($text)
+    {
+        return floor(str_word_count(strip_tags($text)) / 200);
```

Comment:
WP plugin seems to use 250 words per minutes ([see here](https://plugins.trac.wordpress.org/browser/estimated-post-reading-time/trunk/plugin.php#L73)), maybe we can adjust it ?
Also, it could be interesting to explain the calculation, at least what is this number `200`.